x-pack/plugins/encrypted_saved_objects/server/crypto/encrypted_saved_objects_service.ts

```diff
@@ -299,7 +295,7 @@ export class EncryptedSavedObjectsService {
     }
 
     if (Object.keys(attributesAAD).length) {
```

Comment:
I know you didn't change this here, but is this check correct?
```suggestion
     if (!Object.keys(attributesAAD).length) {
```

Reply:
Hmmm, good catch! It looks like :man_facepalming: on my side, let me check.